### DISCUSSION TITLE
Tooltips for the buttons (play aso)

### DISCUSSION THREAD

**Discussion by Travelbacon** [neffo/bing-wallpaper-gnome-extension#186] [Ideas]
First of all, I really like this extension. Thank you for your hard work.

I have some issues understanding the play, forward, pause a.s.o buttons. It is not clear what they exactly do. Is it possible to add a tool tip to them?

**Comment by neffo** [comment]
It is a bit of a limitation of the St toolset used for menus (at the moment at least), but it is a great idea (and thanks for suggestion).

Definitely agree, the icons are not super intuitive. I am considering putting some better documentation here (in wiki perhaps), but this I hope helps:

![Screenshot from 2022-10-03 22-50-14](https://user-images.githubusercontent.com/5569112/193582198-8096af86-8de0-45c8-a6de-cebd336dda01.png)

⏪ - select previous day's image (in date order)
🔁 - select a random image (from downloaded & saved collection only)
⏩ - select next day's image (in date order)
⏭️ - select today's image (skip to current)
➡️ (actually two parallel arrows in GNOME) / 🔀 - toggle shuffle (selects random wallpaper at set interval) **or** sequential mode (default)
▶️ / ⏸️ - toggle between always set new image (default) **or** lock to selected image (image can then be selected from the gallery or through buttons)

- Clicking the thumbnail will open the image in an image viewer.
- Clicking the text description below the image will take you to the Bing description of the image aka [this page](https://www.bing.com/search?q=isle+of+skye+wiki&form=hpcapt&filters=HpDate%3a%2220221003_0700%22) for the image in the screenshot

![Screenshot from 2022-10-03 23-09-07](https://user-images.githubusercontent.com/5569112/193585464-d4b17516-d16b-4950-b5c4-28df41c3b4f5.png)

The 4 buttons in the gallery (4th page in the preferences) do have tool-tips but these do the following:
- Apply - set this image as wallpaper
- View - open image in image viewer
- Info - open the Bing description of the image
- Delete - delete the image from the folder (permanently if it's more than a week old)

**Comment by Travelbacon** [comment]
Thanks that helps. You already wrote the documentation ;)
